api/client/connection.go

```diff
@@ -432,6 +432,25 @@ func (c *Connection) Vacuum(ctx context.Context, pool, revision string, dryrun b
 	return res, err
 }
 
+func (c *Connection) VectorPost(ctx context.Context, pool, revision string, objectIDs []ksuid.KSUID, message api.CommitMessage) (api.CommitResponse, error) {
+	return c.vector(ctx, pool, revision, objectIDs, message, http.MethodPost)
+}
+
+func (c *Connection) VectorDelete(ctx context.Context, pool, revision string, objectIDs []ksuid.KSUID, message api.CommitMessage) (api.CommitResponse, error) {
```

Comment:
Call these AddVectors and DeleteVectors so they match the lake/api.Interface? That's what we've done with the all the other methods.